### DISCUSSION TITLE
Issued the same credential twice

### DISCUSSION THREAD

**Discussion by mdcoxmt** [learningeconomy/LearnCard#190] [Help]
I issued & claimed the same credential twice to the same LearnCard. When I go to view that credential on the LearnCard site, it takes me to a blank page and I need to refresh my browser. 

Any advice on how to mitigate? I would love to be able to either view both credentials, or be warned before I claim a duplicate credential that it could cause problems. 

Also want to figure out how to clear the duplicate so I can view my credential again. 

Before clicking the duplicated credential:
<img width="1472" alt="Screen Shot 2022-11-15 at 12 10 29 PM" src="https://user-images.githubusercontent.com/10966376/202005122-0c3153e9-04e5-48d3-83fc-cea715bf9b45.png">

After clicking the duplicated credential: 
<img width="1473" alt="Screen Shot 2022-11-15 at 12 10 40 PM" src="https://user-images.githubusercontent.com/10966376/202005141-3c239cf7-962d-4a97-89b8-62c0976549f7.png">

**Comment by Custard7** [comment]
Thanks @mdcoxmt for the bug report 🐛

@goblincore @gerardopar we should add this as an issue to our Jira 💪

**Reply by goblincore** [reply]
@mdcoxmt I will look into it

**Reply by goblincore** [reply]
@mdcoxmt Can you provide a screen recording of you creating a duplicate credential and the above happening / clear steps for reproduction of the described situation
